src/app.js

```diff
@@ -1,22 +1,31 @@
+require('dotenv').config();
 const express = require('express');
-
 const router = require('./routes');
 const tutorRouter = require('./routes/tutor');
 const questionRouter = require('./routes/question');
+const authRouter = require('./routes/auth');
+const connectToMongo = require('./db/connection');
+
+const app = express();
+const port = process.env.NODE_LOCAL_PORT;
+
+app.set('view engine', 'ejs');
+
+app.use(express.json());
+
 app.use('/', router);
 app.use('/tutors', tutorRouter);
 app.use('/questions', questionRouter);
+app.use('/auth', authRouter);
 
-require('dotenv').config();
-const connectToMongo = require('./db/connection');
-const app = express();
-const port = process.env.NODE_LOCAL_PORT;
+app.get('/login', (req, res) => {
+  res.render('login');
+});
 
```

Comment:
I think you should move this route to auth/routes file.

Reply:
Thanks, I did it